src/Domain/CoordinateMaps/Wedge.hpp

```diff
@@ -523,6 +525,264 @@ struct WedgeCoordOrientation<3> {
+ *       \end{bmatrix}
+ * \end{align}
+ *
+ * A common factor that shows up in this inverse Jacobian is:
+ *
+ * \begin{align}
+ *   T:= \frac{S(\zeta)}{(\partial_{\zeta}z_{\Lambda})\rho^3}
+ * \end{align}
+ *
+ * And the inverse Jacobian is then:
+ *
+ * \begin{align}
+ *   J^{-1} =
+ *       \frac{1}{z_{\Lambda}}\begin{bmatrix}
+ *         \Xi'^{-1} & 0 & -\rho_x(\Xi'\rho_z)^{-1} \\
+ *         0 & \mathrm{H}'^{-1} & -\rho_y(\mathrm{H}'\rho_z)^{-1} \\
+ *         T\rho_x & T\rho_y &
+ *             T\rho_z + F(\partial_{\zeta}z_{\Lambda}\rho_z)^{-1}/\sqrt{3}
+ *       \end{bmatrix}
+ * \end{align}
+ *
+ * ### Offsetting a Rotated Wedge
+ * The default Wedge map is oriented in the $+z$ direction, so the
+ * construction of a Wedge oriented along a different direction requires an
+ * additional OrientationMap $R$ to be passed to `orientation_of_wedge`. When
+ * offsetting a rotated Wedge, the coordinates passed as parameters to
+ * `focal_offset` are in the coordinate frame in which the Wedge is rotated
+ * (the target frame). However, the focal lifting procedure (shown in
+ * Eq. ($\ref{eq:focal_lifting}$)) is done in the default frame in which the
+ * Wedge is facing the $+z$ direction, so the focal offset $\vec{x}_0$ is first
+ * hit by the inverse rotation $R^{-1}$ and then the rotated focus
+ * $R^{-1}\vec{x}_0$ is used internally as the focus for the $+z$ Wedge. When
+ * the focal lifting calculation has completed, the rotation of the $+z$ Wedge
+ * into the desired orientation by $R$ also rotates the focus into the desired
+ * location. When performing the inverse operation, the focus is similarly
+ * rotated into the default frame, where the inversion is performed.
+ *
+ * ### Interaction between opening angles and focal offsets
+ * When a Wedge is created with a non-zero focal offset, the resulting shape
+ * can take on a variety of possible angular sizes, depending on where the
+ * focus is placed relative to the default centered location. The reader might
+ * note that the angular size of a Wedge can also be controlled by passing an
+ * argument to the `opening_angles` parameter in the Wedge constructor. While
+ * both of these methods allow the angular size of a Wedge to be changed, the
+ * user is prevented from employing both of them at the same time. In
+ * particular, when the the offset is set to some non-zero value, the
+ * `opening_angles_` member variable is set to $\pi/2$. Note that the
+ * `opening_angles_` member being set to $\pi/2$ does not imply the
+ * resulting Wedge will have an angular size of $\pi/2$. On the contrary, the
+ * Wedge will have the angular size that is determined by the application of
+ * the focal lifting method on the parent surface, which is the upper $+z$ face
+ * of a cube that is centered at the origin.
+ *
+ * Because `opening_angles_` is set to $\pi/2$ when there is a non-zero focal
+ * offset, when there is a non-zero focal offset and `with_equiangular_map_` is
```

Comment:
I ended up opting to leave as is since we use language like this in other places since someone might pass a focal offset with zeros, which reduces to a centered Wedge